pkg/skaffold/deploy/kustomize.go

```diff
@@ -123,8 +124,8 @@ func (k *KustomizeDeployer) Deploy(ctx context.Context, out io.Writer, builds []
 
 	namespaces, err := manifests.CollectNamespaces()
 	if err != nil {
-		event.DeployInfoEvent(fmt.Errorf("could not fetch deployed resource namespace."+
-			"This might cause port-forward and deploy health-check to fail: %w", err))
+		event.DeployInfoEvent(fmt.Errorf("could not fetch deployed resource namespace. "+
+			"This might cause port-forward and deploy health-check to fail :%w", err))
```

Comment:
This space move doesn't seem right
```suggestion
			"This might cause port-forward and deploy health-check to fail: %w", err))
```